docs/programmatic-api.md

```diff
@@ -317,6 +320,8 @@ TestCollection API:
 
 * `eachRootSuite((root, browserId) => ...)` - iterates over all root suites in collection which have some tests.
 
+* `format(formatterType)` - formats the tests in one of the available formatting types (`list` or `tree`). You can read more about the available formatting types [here](./cli.md#--formattername).
```

Comment:
User can format test not only via cli command but also using `testCollection.format` method